fapi-client/src/main/scala/com/gu/facia/api/models/collectionconfig.scala

```diff
@@ -1,12 +1,13 @@
 package com.gu.facia.api.models
 
-import com.gu.facia.client.models.{Backfill, CollectionConfigJson}
+import com.gu.facia.client.models.{Backfill, CollectionConfigJson, Metadata}
 
 case class Groups(groups: List[String])
 
 case class CollectionConfig(
     displayName: Option[String],
     backfill: Option[Backfill],
+    metadata: Option[List[Metadata]],
```

Comment:
Why is this `List` of `Metadata`? I thought the original idea was it to be categorised as one of them.